dds/DCPS/TypeSupportImpl_T.h

```diff
@@ -39,7 +39,7 @@ namespace OpenDDS {
     {
       typedef DataWriterImpl_T<MessageType> DataWriterImplType;
 
-      DataWriterImplType* writer_impl;
+      DataWriterImplType* writer_impl = 0;
```

Comment:
This doesn't seem to be necessary since ACE_NEW_RETURN will always assign a value, writer_impl will never be uninitialized.  I don't mind leaving the change in though.